brave/brave-core
# Sites can now be restored from Auto-Contribute box

Fixes brave/brave-browser#1883

Brave UI Implementation: brave/brave-ui#268

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:

1. Start Brave and enable Rewards
2. Visit 3 sites
3. Exclude a site
4. Verify that site can be restored from auto-contribute box
5. Visit 4 to 5 more sites
6. Exclude 1 site
7. Verify that only 'View all sites remain visible
8. Exclude more sites so that only 5 sites are included
9. Verify that sites can be restored from auto-contribute box when only 5 sites remain
10. With all sites restored, click 'See all sites'
11. Exclude sites so that more than 5 exist and close 'See all sites' modal
12. Verify that only 'See all sites' remain visible.
13. Click 'See all sites' again and exclude more sites to get total number include to 5 or less
14. Close 'See all sites' modal and verify that sites can be restored from auto-contribute box


## Reviewer Checklist:
